### PR TITLE
added vertical resize and space holder for presentation slides

### PR DESCRIPTION
**Pull request by UmaMoiseenko**
Reference issue in ng2-codelab https://github.com/kirjs/ng2-codelab/issues/51

Added vertical resize and dropped a spceholder for presentation slides

Also add mini-slides!!
c3kkxc3a8ko

screen shot 2017-04-17 at 9 02 17 pm

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Please merge conflicts

**Comment by kirjs** [comment]
As Uma is travelling, someone will have to pick this up

**Comment by kirjs** [comment]
PTAL

**Comment by kirjs** [comment]
c3kkxc3a8ko

File: src/app/resize/resize.component.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+	<div [class.overlay]="isMouseDown"></div>
+	<div
+		class="spacer"
+		[class.spacer--vertical]="isVertical">
+		<div
+			class="handle"
+			[class.is-dragged]="isMouseDown">
+		</div>
+	</div>
```
Comment (prestonvanloon) [inline src/app/resize/resize.component.html]: This file is formatted oddly. Please fix.

**Pull request merged**